src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -66,6 +66,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

Comment:
Not sure if "call an event" is the best collocation. Maybe "to do it", which will refer to sending an event from the previous sentence?

Reply:
You are right; it sounds like a purple pose. I will change it.